src/components/2FA/index.tsx

```diff
@@ -67,42 +66,50 @@ export const User2FAComponent: FC<Props> = ({
     }
   }, [isSuccess, onSuccessCallback, data])
 
-  const handleSubmit = () => submit2faData({ code, userId })
+  const handleSubmit = () => submit2faData({ secure2FACode, userId })
+
+  const handleKeyDown = (e: React.KeyboardEvent) => {
+    if (secure2FACode.length === 6 && e.code === KeysCodes.ENTER) {
+      e.preventDefault()
+      handleSubmit()
+    }
+  }
 
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (/^[0-9]{0,6}$/.test(e.target.value) || !e.target.value) {
-      setCode(e.target.value)
+    if (digitInputObserver(e.target.value, 6) || !e.target.value) {
+      setSecure2FACode(e.target.value)
     }
 
     e.preventDefault()
   }
 
   return (
     <User2FAComponentBlock>
-      <QrCodeWrapper>
-        <img src={qrcode} />
-      </QrCodeWrapper>
-      <Paragraph>Please scan qrcode and enter your code</Paragraph>
+      <Paragraph>Please enter code from your Google Authentificator </Paragraph>
       <User2FAWrapper>
-        <User2FAInput type='text' onChange={handleChange} value={code} autoFocus />
-        <div>
-          {isLoading ? (
-            <LittleLoader />
-          ) : (
-            <User2FAButton onClick={handleSubmit} disabled={code.length < 6 || isSuccess}>
-              Send
-            </User2FAButton>
-          )}
-        </div>
+        <User2FAInput
+          type='text'
+          onChange={handleChange}
+          onKeyDown={handleKeyDown}
+          value={secure2FACode}
+          autoFocus
+        />
+        {isLoading ? (
+          <LittleLoader />
```

Comment:
Little loader? should this name be changed?
To default loader, or desktop loader or any others which describes anything about this loader

Reply:
Little loader was changed on Loader